defaults/main.yml

```diff
@@ -15,3 +15,11 @@ memcached_cfg_file: '/etc/memcached.conf'
 memcached_consul_tags_default: ['memcached', 'cache']
 memcached_consul_tags_extra: []
 memcached_consul_tags: '{{ memcached_consul_tags_default + memcached_consul_tags_extra }}'
+
+memcached_exporter_enabled: true
+memcached_exporter_version: '0.8.0'
+memcached_exporter_release_name: 'memcached_exporter-{{ memcached_exporter_version }}.linux-amd64'
+memcached_exporter_url: 'https://github.com/prometheus/memcached_exporter/releases/download/v{{ memcached_exporter_version }}/{{ memcached_exporter_release_name }}.tar.gz'
+memcached_exporter_checksum: '0a666f41d733ebdc20cf2dc66fbb0c2899afc237002db67b7a31ff7f9f7dde3f'
+memcached_exporter_group: 'memcached_exporter'
+memcached_exporter_user: 'memcached_exporter'
```

Comment:
Do we really need a separate user for this service? Why not just use the `memcached_user` value`? And the groups seems unnecessary too.

Reply:
I've removed them